### ISSUE TITLE
Error in setting up development environment

### ISSUE THREAD

**Issue by Oyedare** [doczjs/docz#1731]
I'm trying to contribute to the project but when i run yarn dev, it throws an error in the terminal
![Screenshot (11)](https://user-images.githubusercontent.com/59839603/169141900-e71e40f2-c8d7-4dd8-add6-de3e7864ac0d.png)
.

**Comment by Psypher1** [comment]
So it's not just me facing this

**Comment by Oyedare** [comment]
Yeah. I don't know if this problem can be solved, i wish i could tag the owner of the repo but don't know their tag.

**Comment by GeorgianStan** [comment]
I'm also facing issues with the setup. At first, it required typescript to be installed, which is not mentioned in the guide.
Second when I run `npm run docz:dev` I receive the following error **Error [ERR_REQUIRE_ESM]: Must use import to load ES Module**

**Comment by Psypher1** [comment]
A workaround I found while doing something unrelated is to forgo the instructions, copy the packages into your `package.json` , run install. Then hope and pray it works

```json
"docz": "latest",
"prop-types": "^15.8.1"
```
